editor/editor_inspector.cpp

```diff
@@ -2696,638 +2867,656 @@ void EditorInspector::update_tree() {
+						} else {
+							break;
+						}
 					}
-				}
 
-				if (ClassDB::class_exists(classname)) {
-					doc_info_cache[classname][propname] = doc_info; // Do not cache the doc information of scripts.
+					if (ClassDB::class_exists(classname)) {
+						doc_info_cache[classname][propname] = doc_info; // Do not cache the doc information of scripts.
+					}
 				}
 			}
-		}
 
-		Vector<EditorInspectorPlugin::AddedEditor> editors;
-		Vector<EditorInspectorPlugin::AddedEditor> late_editors;
+			Vector<EditorInspectorPlugin::AddedEditor> editors;
+			Vector<EditorInspectorPlugin::AddedEditor> late_editors;
 
-		// Search for the inspector plugin that will handle the properties. Then add the correct property editor to it.
-		for (Ref<EditorInspectorPlugin> &ped : valid_plugins) {
-			bool exclusive = ped->parse_property(object, p.type, p.name, p.hint, p.hint_string, p.usage, wide_editors);
+			// Search for the inspector plugin that will handle the properties. Then add the correct property editor to it.
+			for (Ref<EditorInspectorPlugin> &ped : valid_plugins) {
+				bool exclusive = ped->parse_property(object, p.type, p.name, p.hint, p.hint_string, p.usage, wide_editors);
 
-			for (const EditorInspectorPlugin::AddedEditor &F : ped->added_editors) {
-				if (F.add_to_end) {
-					late_editors.push_back(F);
-				} else {
-					editors.push_back(F);
+				for (const EditorInspectorPlugin::AddedEditor &F : ped->added_editors) {
+					if (F.add_to_end) {
+						late_editors.push_back(F);
+					} else {
+						editors.push_back(F);
+					}
 				}
-			}
 
-			ped->added_editors.clear();
+				ped->added_editors.clear();
 
-			if (exclusive) {
-				break;
+				if (exclusive) {
+					break;
+				}
 			}
-		}
 
-		editors.append_array(late_editors);
+			editors.append_array(late_editors);
 
-		for (int i = 0; i < editors.size(); i++) {
-			EditorProperty *ep = Object::cast_to<EditorProperty>(editors[i].property_editor);
-			const Vector<String> &properties = editors[i].properties;
+			for (int i = 0; i < editors.size(); i++) {
+				EditorProperty *ep = Object::cast_to<EditorProperty>(editors[i].property_editor);
+				const Vector<String> &properties = editors[i].properties;
 
-			if (ep) {
-				// Set all this before the control gets the ENTER_TREE notification.
-				ep->object = object;
+				if (ep) {
+					// Set all this before the control gets the ENTER_TREE notification.
+					ep->object = object;
 
-				if (properties.size()) {
-					if (properties.size() == 1) {
-						//since it's one, associate:
-						ep->property = properties[0];
-						ep->property_path = property_prefix + properties[0];
-						ep->property_usage = p.usage;
-						//and set label?
-					}
-					if (!editors[i].label.is_empty()) {
-						ep->set_label(editors[i].label);
-					} else {
-						// Use the existing one.
-						ep->set_label(property_label_string);
-					}
+					if (properties.size()) {
+						if (properties.size() == 1) {
+							//since it's one, associate:
```

Comment:
```suggestion
							// Since it's one, associate:
```

Reply:
This is out of the scope of my changes, I just placed an `if` enclosing it, it's only "white space changes".